### PR TITLE
Create v1.1.0 Azure Kfdef manifests

### PR DESCRIPTION
**Pull request by eedorenko**
- KFDef manifest for v1.1 to install on Azure
- Anonymous envoy filter for Istio-1.6.x

### CODE REVIEW

**Comment by eedorenko** [comment]
@berndverst FYI

**Comment by berndverst** [comment]
/assign @yanniszark

This is fine from an Azure perspective

File: argo/base_v3/kustomization.yaml [hunk 1]
```diff
@@ -17,10 +17,10 @@ commonLabels:
 images:
 - name: argoproj/argoui
   newName: argoproj/argoui
-  newTag: v2.3.0
+  newTag: v2.4.3
 - name: argoproj/workflow-controller
   newName: argoproj/workflow-controller
-  newTag: v2.3.0
+  newTag: v2.7.5
```
Comment (yanniszark) [inline argo/base_v3/kustomization.yaml:23]: Is this change necessary for Azure?
I'm a bit reluctant to make this change in this PR.
How is this going to affect Kubeflow Pipelines users? Is Pipelines compatible with this Argo version?
I think we should split this into a separate PR and validate compatibility.

**Review by yanniszark — CHANGES_REQUESTED** [review]
Hi @eedorenko! The PR overall looks good, only a minor change.
I would suggest also the following:
- Add the people responsible for manifests on Azure's side to `stacks/azure/OWNERS` and `kfdef/OWNERS`.
- Update the website instructions accordingly, if necessary.

File: argo/base_v3/kustomization.yaml (hunk 1 above)
Comment (eedorenko) [reply argo/base_v3/kustomization.yaml]: Ok. I took it out. I will create a separate PR for that

**Comment by eedorenko** [comment]
>Add the people responsible for manifests on Azure's side to stacks/azure/OWNERS and kfdef/OWNERS.
Yes, once they are approved as contributors

>Update the website instructions accordingly, if necessary.
Yep. On our list. Once this PR is merged.

**Comment by eedorenko** [comment]
@yanniszark done!

**Comment by eedorenko** [comment]
The PR to add Azure owners https://github.com/kubeflow/manifests/pull/1524
